### DISCUSSION TITLE
Rare 500 error responses

### DISCUSSION THREAD

**Discussion by daryayermakova** [etsy/open-api#1164] [Q&A]
The API returns 500 errors occasionally with the response
```
{"error":"Server Error"}
```
and no more information

Mainly it happens for:
- get the list of receipts https://developers.etsy.com/documentation/reference#operation/getShopReceipts (using parameters `min_created`, `max_created`, `limit`, `offset`
- refresh access token `https://openapi.etsy.com/v3/public/oauth/token` (plenty of different users)
- get the list of listings, create a listing

It's really a small percent of 500 errors in comparison with other responses (2xx, 4xx), but anyway: what could it be?

**Comment by Gareth064** [comment]
Since its a "Server Error" there is nothing that you can gleam from it to understand why it happens. It would need to be raised with Etsy directly to ask them for an explanation. Email then at developer@etsy.com

When you make requests to the API, every response, including errors like this, should include a header with an Etsy Request ID. If you give that to Etsy in the email, they might be able to trace your request and come up with an answer to why it failed.

Etsy developers don't work from this discussion board, its only for the community to use to help each other.
